internal/operators/ocs/ocs_operator.go

```diff
@@ -92,6 +95,54 @@ func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 }
 
 // GetHostRequirements provides operator's requirements towards the host
-func (o *operator) GetHostRequirements(context.Context, *common.Cluster, *models.Host) (*models.ClusterHostRequirementsDetails, error) {
-	return &models.ClusterHostRequirementsDetails{}, nil
+func (o *operator) GetHostRequirements(_ context.Context, cluster *common.Cluster, host *models.Host) (*models.ClusterHostRequirementsDetails, error) {
+	numOfHosts := len(cluster.Hosts)
+
+	inventory, err := hostutil.UnmarshalInventory(host.Inventory)
+	if err != nil {
+		return nil, err
+	}
+
+	disks := int64(getValidDiskCount(inventory.Disks) - 1)
+
+	if numOfHosts < 3 {
+		return nil, fmt.Errorf("number of hosts is less than 3")
```

Comment:
What will this message look like when it is finally displayed? Will the operator name be included automatically? Otherwise please reword error msg to "OCS require a minimum of 3 hosts".

The same comment holds for all error messages.